### PR TITLE
Grammar update

### PR DESCRIPTION
**Pull request by DocAdam**
'want archive' updated to: want to archive

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

File: content/issues/organizing-your-work-with-project-boards/tracking-work-with-project-boards/archiving-cards-on-a-project-board.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Automation in your project board does not apply to archived project board cards.
 
 ## Archiving cards on a project board
 
-1. In a project board, find the card you want archive, then click {% octicon "kebab-horizontal" aria-label="The horizontal kebab icon" %}.
+1. In a project board, find the card you want to archive, then click {% octicon "kebab-horizontal" aria-label="The horizontal kebab icon" %}.
 ![List of options for editing a project board card](/assets/images/help/projects/select-archiving-options-project-board-card.png)
```
Comment (goncharova7714) [inline content/issues/organizing-your-work-with-project-boards/tracking-work-with-project-boards/archiving-cards-on-a-project-board.md:24]: # See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.177.0/containers/javascript-node/.devcontainer/base.Dockerfile

# [Choice] Node.js version: 16, 14, 12
ARG VARIANT="16-buster"
FROM mcr.microsoft.com/vscode/devcontainers/javascript-node:0-${VARIANT}

# [Optional] Uncomment this section to install additional OS packages.
# RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
#     && apt-get -y install --no-install-recommends <your-package-list-here>

# [Optional] Uncomment if you want to install an additional version of node using nvm
# ARG EXTRA_NODE_VERSION=10
# RUN su node -c "source /usr/local/share/nvm/nvm.sh && nvm install ${EXTRA_NODE_VERSION}"

# [Optional] Uncomment if you want to install more global node modules
# RUN su node -c "npm install -g <your-package-list-here>"

**Comment by adwokat** [comment]
Er

**Review by ramyaparimi — APPROVED** [review]
@DocAdam 
Thanks so much for opening a PR with this fix! I'll get this merged once tests are passing 🍏

**Comment by Manuel13dpl** [comment]
Permiso para acceder a memoria SD storage o emulate de Huawei honor 9xde Manuel😎

**Comment by Manuel13dpl** [comment]
@goncharova7714 @Manuel13dpl `

**Comment by Distruptors** [comment]
> 
> ## Automatically generated comment ℹ️
> **This comment is automatically generated and will be overwritten every time changes are committed to this branch.**
> 
> The table contains an overview of files in the `content` directory that have been changed in this pull request. It's provided to make it easy to review your changes on the staging site. Please note that changes to the `data` directory will not show up in this table.
> 
> ---
> 
> ### Content directory changes
> _You may find it useful to copy this table into the pull request summary. There you can edit it to share links to important articles or changes and to give a high-level overview of how the changes in your pull request support the overall goals of the pull request._
> | **Source** | **Staging** | **Production** | **What Changed** |
> |:----------- |:----------- |:----------- |:----------- |
> | [content/issues/organizing-your-work-with-project-boards/tracking-work-with-project-boards/archiving-cards-on-a-project-board.md](https://github.com/github/docs/blob/214c27b4a1952f465359abad83e5f2250de6200b/content/issues/organizing-your-work-with-project-boards/tracking-work-with-project-boards/archiving-cards-on-a-project-board.md) | [Modified](https://docs-13254--patch-7.herokuapp.com/issues/organizing-your-work-with-project-boards/tracking-work-with-project-boards/archiving-cards-on-a-project-board) | [Original](https://docs.github.com/issues/organizing-your-work-with-project-boards/tracking-work-with-project-boards/archiving-cards-on-a-project-board) | |

**Pull request merged**